docs/roadmap.md

```diff
@@ -18,30 +18,30 @@ clustered database or key-value store. We will target such workloads for our
 
 ## APIs and core features
 1. Consistent v1 API
-  - Status: v1beta3 (#1519) is being developed as the release candidate for the v1 API.
-2. Multi-port services for apps which need more than one port on the same portal IP (#1802)
-  - Status: #2585 covers the design.
-3. Nominal services for applications which need one stable IP per pod instance (#260)
+  - Status: DONE. [v1beta3](http://kubernetesio.blogspot.com/2015/04/introducing-kubernetes-v1beta3.html) was developed as the release candidate for the v1 API.
+2. Multi-port services for apps which need more than one port on the same portal IP ([#1802](https://github.com/GoogleCloudPlatform/kubernetes/issues/1802))
+  - Status: DONE. Released in 0.15.0
+3. Nominal services for applications which need one stable IP per pod instance ([#260](https://github.com/GoogleCloudPlatform/kubernetes/issues/260))
   - Status: #2585 covers some design options.
-4. API input is scrubbed of status fields in favor of a new API to set status (#4248)
-  - Status: in progress
-5. Input validation reporting versioned field names (#2518)
-  - Status: in progress
+4. API input is scrubbed of status fields in favor of a new API to set status ([#4248](https://github.com/GoogleCloudPlatform/kubernetes/issues/4248))
+  - Status: DONE
+5. Input validation reporting versioned field names ([#2518](https://github.com/GoogleCloudPlatform/kubernetes/issues/2518))
```

Comment:
I dont think that this is done yet: https://github.com/GoogleCloudPlatform/kubernetes/issues/3084